contrib/rpm/buildah.spec

```diff
@@ -259,13 +259,13 @@ make DESTDIR=%{buildroot} PREFIX=%{_prefix} install install.completions
 - Turn on --enable-gc when running gometalinter
 - rmi: handle truncated image IDs
 
-* Tue Aug 15 2017 Josh Boyer <jwboyer@redhat.com> - 0.3-5.gitb9b2a8a
+* Tue Aug 15 2017 Josh Boyer <jwboyer@redhat.com> 0.3-5.gitb9b2a8a
 - Build for s390x as well
 
-* Wed Aug 02 2017 Fedora Release Engineering <releng@fedoraproject.org> - 0.3-4.gitb9b2a8a
+* Wed Aug 02 2017 Fedora Release Engineering <releng@fedoraproject.org> 0.3-4.gitb9b2a8a
 - Rebuilt for https://fedoraproject.org/wiki/Fedora_27_Binutils_Mass_Rebuild
 
-* Wed Jul 26 2017 Fedora Release Engineering <releng@fedoraproject.org> - 0.3-3.gitb9b2a8a
+* Wed Jul 27 2017 Fedora Release Engineering <releng@fedoraproject.org> 0.3-3.gitb9b2a8a
```

Comment:
why the date change from 26 to 27?  Was it a cut/paste error?

Reply:
Good catch.. and my answer is... no idea.

I'll sort it out now and re-submit.